RUFAS/biophysical/animal/animal_properties/reproduction_properties.py

```diff
@@ -3,4 +3,20 @@
 
 @dataclass
 class ReproductionProperties:
-    pass
+    """
+    Collection of animal properties that are related to reproduction.
+
+    Attributes
+    ----------
+    gestation_length: int
+    concpeptus_weight: float
+    calf_birth_weight:float
+    calves: int
+    calving_interval: float
```

Comment:
Could descriptions and units (where appropriate) be added to this docstring?
```suggestion
    gestation_length: int
    concpeptus_weight: float
    calf_birth_weight: float
    calves: int
    calving_interval: float
```